app/scripts/filters/resources.js

```diff
@@ -320,7 +320,7 @@ angular.module('openshiftConsole')
         return routeWebURLFilter(route, host);
       }
       var label = (host || routeHostFilter(route));
```

Comment:
If we don't want the path to show up, I'd just return early here:

```js
var label = (host || routeHostFilter(route));
if (!label) {
  return '<unknown host>';
}
```

but I have a slight preference for showing the path.